client/src/sass/application.scss

```diff
@@ -383,6 +384,10 @@ table {
       .admin-sub-header {
         @include admin-sub-header-responsive($menu-width/2 + $expanded-horizontal-margins/3);
       }
+
+      .sub-menu {
+        overflow-x: auto;
+      }
```

Comment:
I think we can delete this one, when the left menu is opened, the dropdowns of the the user-menu will not be replaced by a modal with a screen width higher than 800px (cause handling this with the screenService was a bit tricky compared to css).

In any case, for that kind of small device we can lock the scroll to show dropdown ? the user will hide the left menu to see the rest of the menu ?